appointments.php

```diff
@@ -7,20 +7,59 @@
  * Author: Harrisong Gutierrez
  */
 
-
-
-
 if (!defined('ABSPATH')) {
     exit;
 }
 
+
 define('APPOINTMENTS_PLUGIN_PATH', plugin_dir_path(__FILE__));
 define('APPOINTMENTS_PLUGIN_URL', plugin_dir_url(__FILE__));
 
-require_once APPOINTMENTS_PLUGIN_PATH . 'admin/inc/admin-functions.php';
-require_once APPOINTMENTS_PLUGIN_PATH . 'appointments/inc/appointments-functions.php';
-require_once APPOINTMENTS_PLUGIN_PATH . 'admin/inc/admin-menu.php';
 
+require_once APPOINTMENTS_PLUGIN_PATH . 'admin/inc/AdminClass.php';
+require_once APPOINTMENTS_PLUGIN_PATH . 'appointments/inc/AppointmentsClass.php';
+
+class Appointments_Plugin
+{
+
+    public function __construct()
+    {
+
+        register_activation_hook(__FILE__, [$this, 'create_appointments_tables']);
+        register_deactivation_hook(__FILE__, [$this, 'drop_appointments_tables']);
+
+        add_action('admin_enqueue_scripts', [$this, 'enqueue_admin_scripts']);
+
+
+        new Admin_Class();
```

Comment:
inicializar correctamente o eliminarlo si no se esta utilizando

Reply:
This part of the code is currently functional